test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1222,6 +1121,28 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
 
     #endregion Content Header Inclusion
 
+    #region Client Certificate Authentication
+
+    It "Verifies Invoke-WebRequest Certificate Authentication Fails without -Certificate"  {
+        $uri = 'https://localhost:8083/'
```

Comment:
I believe we should create a new functions (Get-HttpsTestUrl/Get-HttpTestUrl?) in new HttpsListener module to get the test server Urls .